java/src/test/java/org/whispersystems/libsignal/ecc/Curve25519Test.java

```diff
@@ -10,7 +10,7 @@
 public class Curve25519Test extends TestCase {
 
   public void testAgreement() throws InvalidKeyException {
-    byte[] alicePublic  = {(byte) 0x05, (byte) 0x1b, (byte) 0xb7, (byte) 0x59, (byte) 0x66,
```

Comment:
Various tests had hardcoded keys in them. Since we now longer have the 1 byte key type prefix, I updated the hardcoded keys to omit these. You'll see this in various places.